src/main/kotlin/lotto/domain/NumberGenerator.kt

```diff
@@ -1,11 +1,11 @@
 package lotto.domain
 
-object RandomNumbers {
+object NumberGenerator {
     fun get(): Set<Int> {
         return (LottoNumber.MIN_RANGE_NUMBER..LottoNumber.MAX_RANGE_NUMBER)
             .shuffled()
             .take(LottoTicket.SUFFICIENT_SIZE)
             .sorted()
             .toCollection(linkedSetOf())
     }
```

Comment:
https://github.com/herotechcourse/kotlin-lotto/pull/28#discussion_r2195200863
Where is the test code ?! 👀 
I think u can generate test code for this wouldn’t be too hard. 😄